README.md

```diff
@@ -1,70 +1,141 @@
+To get a local copy up and running follow these simple example steps.
 
-### Making a Progressive Web App
+- Open terminal
+- Clone this project by the command `https://github.com/TaiwoEnoch/cryptocurrency-data-explorer.git`
+- `cd <clone>` folder
+- Run `npm start` in your local browser or using Live Server in Visual Studio Code.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+### Prerequisites
 
-### Advanced Configuration
+- IDE to edit and run the code (We use Visual Studio Code 🔥).
+- [Node.js](https://nodejs.org/en/download/) already downloaded.
+- Create React App - [click here to getting started](https://create-react-app.dev/docs/getting-started).
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+### Setup
 
-### Deployment
+Clone this repository to your desired folder:
+`cd my-folder`
+`git clone https://github.com/TaiwoEnoch/cryptocurrency-data-explorer.git`
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+### Usage
+To run the project execute the command below 
 
-### `npm run build` fails to minify
+`npm start` 
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+### Run tests
+
+To run tests, run the following command:
+
+- npx eslint "\*_/_.{js,jsx}"
+
+### Future Features
+
+- Create an interactive web application using React and Redux that allows users to explore Cryptocurrencies data in a variety of currencies.
+- Add a crypto news feed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author
+ <a name="authors"></a>
+
+👤 **Enoch Taiwo**
+
+- [Github](https://github.com/TaiwoEnoch)
+- [Linkedin](https://www.linkedin.com/in/taiwo01/)
+- [Twitter](https://twitter.com/taiwoenoch4)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can give it a ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Hat tip to [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio) as his design was used 🔰
+- The standup team 🏹⚡
+- My Family 🙌
+- Kudos to my coding partners
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly fix your license link in your Readme file. :+1: Currently the link to your license is a broken link as you can see below :point_down: 
  
  ![image](https://user-images.githubusercontent.com/92303891/236536834-6e6a9911-850d-4194-80a2-4974bbf8789a.png)